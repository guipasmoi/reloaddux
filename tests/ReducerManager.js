describe("combineReducerTree", () => {
  it("It can init state with init action", () => {
    // const toto = "";
  });
});
