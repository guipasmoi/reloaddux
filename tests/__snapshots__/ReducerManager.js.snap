// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReducerManager it can register multiple tree reducers 1`] = `
Object {
  "counters": Object {
    "Awesome": Object {
      "StateFor": Object {
        "Counter3": undefined,
      },
    },
    "AwesomeStateForCounter1": 12,
    "AwesomeStateForCounter2": 24,
  },
}
`;

exports[`ReducerManager it can register multiple tree reducers 2`] = `
Object {
  "counters": Object {
    "Awesome": Object {
      "StateFor": Object {
        "Counter3": undefined,
      },
    },
    "AwesomeStateForCounter1": 12,
    "AwesomeStateForCounter2": 26,
  },
}
`;

exports[`ReducerManager it can register multiple tree reducers 3`] = `
Object {
  "counters": Object {
    "Awesome": Object {
      "StateFor": Object {
        "Counter3": 36,
      },
    },
    "AwesomeStateForCounter1": 12,
    "AwesomeStateForCounter2": 28,
  },
}
`;

exports[`ReducerManager it can unregister tree reducers 1`] = `
Object {
  "counters": Object {
    "Awesome": Object {
      "StateFor": Object {
        "Counter3": undefined,
      },
    },
    "AwesomeStateForCounter1": 12,
    "AwesomeStateForCounter2": 24,
  },
}
`;

exports[`should not have change after action 1 1`] = `
Object {
  "counters": Object {
    "Awesome": Object {
      "StateFor": Object {
        "Counter3": undefined,
      },
    },
    "AwesomeStateForCounter1": 12,
    "AwesomeStateForCounter2": 26,
  },
}
`;
